catalog/update.go

```diff
@@ -29,8 +29,22 @@ func updateFunctionFactory(pathPrefix string, updatable []string) func(context.C
 			}, field) {
 				continue
 			}
-			old, new := d.GetChange(field)
-			if !d.HasChange(field) {
+
+			var old, new interface{}
+			if field == "columns" {
```

Comment:
wouldn't it be simpler to do:

```
if field == "columns" {
  field = "column"
}
old, new = d.GetChange(field)
if !d.HasChange(field) {
  continue
}

Reply:
the API expect `columns` in the payload however